wom/src/main/scala/wom/types/coercion/defaults/package.scala

```diff
@@ -0,0 +1,29 @@
+package wom.types.coercion
+
+import cats.syntax.validated._
+import common.validation.Validation._
+import common.validation.ErrorOr.ErrorOr
+import common.validation.ErrorOr._
+import wom.types._
+import wom.values._
+
+import scala.language.implicitConversions
+import scala.reflect.ClassTag
+
+package object defaults {
+  implicit val womIntegerCoercer: WomTypeCoercer[WomInteger] = defaultCoercionForType[WomInteger](WomIntegerType)
+  implicit val womFloatCoercer: WomTypeCoercer[WomFloat] = defaultCoercionForType[WomFloat](WomFloatType)
+  implicit val womStringCoercer: WomTypeCoercer[WomString] = defaultCoercionForType[WomString](WomStringType)
```

Comment:
The default coercers tie together the `WomValue` type and the coercion function on their respective `WomType`s